modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -285,13 +209,26 @@ If you do not use a default Red Hat NTP server, verify the NTP server for your p
 `inputs13.osdsecuritylogs.splunkcloud.com`
 `inputs14.osdsecuritylogs.splunkcloud.com`
 `inputs15.osdsecuritylogs.splunkcloud.com`
-|Required. Used by the `splunk-forwarder-operator` as a log forwarding endpoint to be used by Red Hat SRE for log-based alerting.
 |9997
+|Used by the `splunk-forwarder-operator` as a logging forwarding endpoint to be used by Red Hat SRE for log-based alerting.
 
-|`observatorium.api.openshift.com`
-|Required. Used for Managed OpenShift-specific telemetry.
+|`http-inputs-osdsecuritylogs.splunkcloud.com`
 |443
+|Required. Used by the `splunk-forwarder-operator` as a logging forwarding endpoint to be used by Red Hat SRE for log-based alerting.
+
+|`sftp.access.redhat.com` (Recommended)
+|22
+|The SFTP server used by `must-gather-operator` to upload diagnostic logs to help troubleshoot issues with the cluster.
 |===
+
+. If you did not allow a wildcard for Amazon Web Services (AWS) APIs, you will need to also allow the S3 bucket used for the internal OpenShift registry. To retrieve that endpoint, run the following command once the cluster has successfully been provisioned:
```

Comment:
Should OpenShift here be {product-title}?

Reply:
This should not be {product-title}, as it is referencing OpenShift not ROSA.

Reply:
Thanks for clarifying @michaelryanmcneill!